feat: void element tags in helpers are selectable between `>` and `/>` (#6789)
codeigniter4/CodeIgniter4: user_guide_src/source/installation/upgrade_430.rst

Breaking Enhancements
*********************

> - Since void HTML element (e.g. ``<input>``) in ``html_helper``, ``form_helper`` or common functions has been changed to HTML5 valid by default. If you need to be compatible with XHTML, set the ``$html5`` property in **app/Config/DocTypes.php** to ``false``.

Reviewer: ```suggestion
- Since void HTML elements (e.g. ``<input>``) in ``html_helper``, ``form_helper`` or common functions have been changed to be HTML5-compatible by default and you need to be compatible with XHTML, you must set the ``$html5`` property in **app/Config/DocTypes.php** to ``false``.
```

Author: Done

The following files received significant changes (including deprecations or visual adjustments)
and it is recommended that you merge the updated versions with your application:

* ``app/Config/DocTypes.php``

>     * The property ``$html5`` to determine whether to remove the solidus (``/``) character for void HTML elements (e.g. ``<input>``) is added, and set true for HTML5 compatibility by default.

Reviewer: ```suggestion
    * The property ``$html5`` to determine whether to remove the solidus (``/``) character for void HTML elements (e.g. ``<input>``) is added, and set to ``true`` by default for HTML5 compatibility.
```

Author: Done
